.github/workflows/ciab.yaml

```diff
@@ -69,6 +70,7 @@ on:
       - 'NOTICE'
       - 'traffic_control/java/**'
       - 'cache-config/testing/**'
+      - 'cache-config/testing/docker/trafficserver**'
```

Comment:
Since this section is `on.pull_request.paths-ignore`, `cache-config/testing/docker/trafficserver/**` should be prefixed with `!` to register the exception.

Reply:
reverted, update ciab.yaml in another PR.